dashboard/settings.py

```diff
@@ -70,7 +71,7 @@
 # See https://docs.djangoproject.com/en/1.9/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = ENV.get('DJANGO_SECRET_KEY')
+SECRET_KEY = ENV.get('DJANGO_SECRET_KEY', get_random_secret_key())
```

Comment:
Cool, didn't know this function existed. Doesn't seem to be well documented. I wonder if there would be any drawback/risk to using this with deployments instead of specifying it ourselves.

Reply:
I think the only problem with having it random in production would be if the server reset some tokens may not work.

And if was run with multiple pods, they'd all get a different random key, and not sure if this would cause any issues or not with the setup we have. We'd have to test if we want to try it.